.github/workflows/verify-claims.yml

```
@@ -0,0 +1 @@
+# throwaway test commit — will be closed without merging
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)